ogc-rs/Cargo.toml

```diff
@@ -10,6 +10,7 @@ name = "ogc"
 
 [dependencies]
 bitflags = "1.2"
+libc = "0.2.94"
```

Comment:
We probably shouldn't use a definite version here, just ``0.2`` should be enough to cover the latest patch versions released without future commits to changing it.

Reply:
Changed to `0.2`.